src/wp-admin/includes/class-walker-nav-menu-edit.php

```diff
@@ -262,6 +262,29 @@ public function start_el( &$output, $data_object, $depth = 0, $args = null, $cur
 					<button type="button" class="button-link menus-move menus-move-top" data-dir="top"><?php _e( 'To the top' ); ?></button>
 				</fieldset>
 
+				<?php
+				/**
+				 * Update parent and order of menu item by changing it.
+				 *
+				 * @since 6.6.0
+				 */
+				?>
+
+				<fieldset class="field-move-combo description description-wide">
```

Comment:
I can see that `fieldset` is used elsewhere without a `legend`, but let's not add another. Change this to a `div`. Properly speaking, we should have one fieldset wrapping all the controls for a given menu item; but I think that should be a separate issue.